spec/spec_helper.rb

```diff
@@ -1,3 +1,4 @@
+require 'database_cleaner'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.